src/StoreKeeper/WooCommerce/B2C/Tools/WooCommerceAttributeMetadata.php

```diff
@@ -126,13 +126,12 @@ public static function setMetadata($attribute_id, $meta_key, $meta_value)
     /**
      * @param $attribute_id
      * @param $meta_key
-     * @param bool $single
      *
      * @return mixed
      *
      * @throws \Exception
      */
-    public static function getMetadata($attribute_id, $meta_key, $single = true)
+    public static function getMetadata($attribute_id, $meta_key, bool $single = true, $order = 'ASC')
```

Comment:
Why the order is needed? Though you can't have duplicates on meta_key

Reply:
>  Though you can't have duplicates on meta_key

Unfortunately, it does not work like that at the moment, it keeps on recreating meta for attributes. See screenshot below:

Reply:
Looks more like bug to me, cannot think of a case that you would like to have double. This table is ours.

Reply:
Created bug card for this:
https://app.clickup.com/t/k01pjv